### PR TITLE
Include change management in Managed Services

### PR DESCRIPTION
**Pull request by kjacobsen**
As Managed Services is responsible for production systems, update wording to not only include agile development but also include change management practices as well.

Also added a line about ensuring there are the appropriate approvals for changes to occur in production.

### CODE REVIEW

File: Managed Services.md [hunk 1]
```diff
@@ -9,13 +9,14 @@
 - I am learning how to write valuable automated tests.
 - I am becoming proficient with my tools and the core building blocks used by our team.
 - I am learning how to pull together the Lego blocks of a product when developing feature enhancements to software products.
+- I do not deploy production changes without the appropriate aprovals.
```
Comment (aaronpowell) [inline Managed Services.md]: This point feels a little specific to me, that there's really only one way to demonstrate the behaviour.

**Comment by kjacobsen** [comment]
@aaronpowell, whilst specific, I feel that this is one of those times that we need to be "oddly specific".

Alternative, and probably better wording might be, 

> I only deploy production changes with the appropriate approvals.

File: Managed Services.md (hunk 1 above)
Comment (andrewabest) [inline Managed Services.md]: Agreed. This is more about following established processes / being a disciplined team member.

**Comment by andrewabest** [comment]
@kjacobsen how about

> I am rigorous and disciplined when deploying to production, and always adhere to appropriate change governance and process

Trying to make it read more like something someone can connect with rather than a warning.

File: Managed Services.md [hunk 2]
```diff
@@ -115,7 +116,7 @@
 - I am prioritising the needs of my team members above my own, knowing that their success is our success.
 - I am an outspoken advocate of my team. They can trust me to be the first line of defense.
 - I know the buck stops with me. At the end of the day I am responsible for my team and the relationship with our client.
-- I actively track the progress of our team using agile project management techniques, keeping the goals of the project and our constraints front of mind. I make this information visible and transparent internally and externally to minimise 'surprises' and to facilitate early and pragmatic problem resolution.
+- I actively track the progress of our team using agile project management techniques and against change control processes, keeping the goals of the project and our constraints front of mind. I make this information visible and transparent internally and externally to minimise 'surprises' and to facilitate early and pragmatic problem resolution.
```
Comment (andrewabest) [inline Managed Services.md]: > I actively track the progress of our team using agile project management techniques and against change control processes

You don't really track progress against change control processes - I think that addition needs to be extracted into a separate point if you want to make another point with it. So far above we have said we embrace change management and are proficient in it. What are we trying to say by adding it here?

**Comment by andrewabest** [comment]
Is this one moving anywhere @kjacobsen? Would be good to get a few of these older PR's merged or closed :shipit:

**Comment by kjacobsen** [comment]
This is now pending the other activities around SC/SE/TL. Certainly cannot close it until the issues are addressed.

**Comment by todthomson** [comment]
@kjacobsen did you want to rebase this one and we'll touch it up a little and get it merged? Or is it still "pending the other activities around SC/SE/TL"?

**Comment by kjacobsen** [comment]
I think the following should be the approach:

1. Close this PR.
2. Create new PR that removes the Sustained roles
3. Ensure that Change Management and similar processes is defined in Dev, SD MadSkillz
4. Finalize SC/SE/TL work.

Does everyone agree?

**Comment by andrewabest** [comment]
@kjacobsen I certainly think closing this PR and opening new ones for your finer points is a good idea 👍 I don't have context around removing the sustained roles.

**Pull request closed without merging**